test/utils/follow-up-claim.js

```diff
@@ -54,7 +59,7 @@ export async function createPreMultipleHerdSheepFollowUp() {
   await verifySubmission("Claim complete");
 }
 
-export async function createPreMultipleHerdPigsFollowUp(sbi, { urn = "pg-fc-5343462" } = {}) {
+export async function createPreMultipleHerdPigsFollowUp({ urn = "pg-fc-5343462" } = {}) {
```

Comment:
```suggestion
export async function createPreMultipleHerdPigsFollowUp(urn = "pg-fc-5343462") {
```

I think this is a little easier to read, and does the same thing